apps/andi/assets/css/edit.scss

```diff
@@ -19,35 +28,105 @@
 }
 
 .edit-button-group {
+  margin-top: 2rem;
+  margin-bottom: 2rem;
```

Comment:
Not trying to be picky, but you could do:
`margin: 2rem auto;`

Reply:
I personally think it is more clear this way, but I will leave this open in case anyone else has an opinion

Reply:
Yeah, I go back and forth on this one. At one point SASS linters (or at least Webstorm) would gripe until you did the one liner. However, I'd say it's fine and possibly more clear to keep it separate. I can never remember that it's T R B L and T(B) R(L) when I need to